velox/functions/FunctionRegistry.cpp

```diff
@@ -124,6 +124,14 @@ std::shared_ptr<const Type> resolveSimpleFunction(
       exec::SimpleFunctions().resolveFunction(functionName, argTypes);
 
   if (resolvedFunction) {
+    if (auto customType = getType(
```

Comment:
@laithsakka We should not need this code, right? Can we make exec::SimpleFunctions().resolveFunction return the correct type?

Reply:
@mbasmanova I improved this fix in my latest commit.